examples/13-vrm/plot_vrm_fwd.py

```diff
@@ -0,0 +1,258 @@
+used to model the VRM response while an analytic solution for a conductive
+half-space is used to model the inductive response.
+"""
+import SimPEG.VRM as VRM
+import numpy as np
+from SimPEG import mkvc, Mesh, Maps
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+
+def run(plotIt=True):
+
+    # CREATE MESH
+    # Works for 3D tensor and 3D tree meshes
+    cs, ncx, ncy, ncz, npad = 2., 35, 35, 20, 5
+    hx = [(cs, npad, -1.3), (cs, ncx), (cs, npad, 1.3)]
+    hy = [(cs, npad, -1.3), (cs, ncy), (cs, npad, 1.3)]
+    hz = [(cs, npad, -1.3), (cs, ncz), (cs, npad, 1.3)]
+    mesh = Mesh.TensorMesh([hx, hy, hz], 'CCC')
+
+    # SET MAPPING AND ACTIVE CELLS
+    # Only cells below the surface (z=0) are being modeled.
+    topoCells = mesh.gridCC[:, 2] < 0.
+
+    # CREATE MODEL (XI: THE AMALGAMATED MAGNETIC PROPERTY)
+    # The model is made by summing a set of 3D Gaussian distributions.
+    # Only active cells have a model value.
+    xyzc = mesh.gridCC[topoCells, :]
+    C = 2*np.pi*8**2
+    xi_true = (
+        4e-4*np.exp(-(xyzc[:, 0]-50)**2/(3*C)) *
+        np.exp(-(xyzc[:, 1])**2/(20*C)) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        4e-4*np.exp(-(xyzc[:, 0]+50)**2/(3*C)) *
+        np.exp(-(xyzc[:, 1])**2/(20*C)) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        4e-4*np.exp(-(xyzc[:, 0]+40)**2/(3*C)) *
+        np.exp(-(xyzc[:, 1]-40)**2/C) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        6e-4*np.exp(-(xyzc[:, 0]+20)**2/C) *
+        np.exp(-(xyzc[:, 1]-10)**2/C) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        8e-4*np.exp(-(xyzc[:, 0]+15)**2/(3*C)) *
+        np.exp(-(xyzc[:, 1]+20)**2/(0.4*C)) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        6e-4*np.exp(-(xyzc[:, 0]-20)**2/(0.5*C)) *
+        np.exp(-(xyzc[:, 1]-15)**2/(0.5*C)) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        8e-4*np.exp(-(xyzc[:, 0]+10)**2/(0.1*C)) *
+        np.exp(-(xyzc[:, 1])**2/(0.1*C)) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        8e-4*np.exp(-(xyzc[:, 0]-25)**2/(0.1*C)) *
+        np.exp(-(xyzc[:, 1])**2/(0.4*C)) *
+        np.exp(-(xyzc[:, 2])**2/C) +
+        1e-5
+        )
+
+    # SET THE TRANSMITTER WAVEFORM
+    # This determines the off-time decay behaviour of the VRM response
+    waveform = VRM.WaveformVRM.StepOff()
+
+    # CREATE SURVEY
+    # Similar to an EM-63 survey by all 3 components of the field are measured
+    times = np.logspace(-5, -2, 31)  # Observation times
+    x, y = np.meshgrid(np.linspace(-30, 30, 21), np.linspace(-30, 30, 21))
+    z = 0.5*np.ones(x.shape)
+    loc = np.c_[mkvc(x), mkvc(y), mkvc(z)]  # Src and Rx Locations
+
+    srcListVRM = []
+
+    for pp in range(0, loc.shape[0]):
+
+        loc_pp = np.reshape(loc[pp, :], (1, 3))
+        rxListVRM = [VRM.Rx.Point(loc_pp, times=times, fieldType='dbdt', fieldComp='z')]
+
+        srcListVRM.append(
+            VRM.Src.MagDipole(rxListVRM, mkvc(loc[pp, :]), [0., 0., 0.01], waveform))
+
+    SurveyVRM = VRM.Survey(srcListVRM)
```

Comment:
sorry to be picky here, but for the examples, code style is important. In python, it is standard to use lower-case for instantiated objects 
`surveyVRM`, `problemVRM`, `fieldsVRM`, etc